### DISCUSSION TITLE
Problem in settings running in inner folder instead of root

### DISCUSSION THREAD

**Discussion by maor365scores** [alirezanet/Husky.Net#116] [Q&A]
Hi all
this is my configs,the Task-runner. the prettier runs ok, but the Run csharpier command is running in the root of the project and not

 in the CWD I specified, this is problematic, need help to solve it pls, I tried relative and absolute path but it does not work, the files which arent in the desired "CWD" folder also get changed by the csharpier

```
{
  "tasks": [
    {
      "name": "Run csharpier",
      "command": "dotnet",
      "pathMode": "absolute",
      "cwd": "Web/CMS/CMS/",
      "args": ["csharpier", "${staged}"],
      "include": ["**/*.cs"]
    },
    {
      "name": "prettier",
      "pathMode": "absolute",
      "cwd": "Web/CMS/CMS/ClientApp/",
      "command": "npx",
      "args": ["prettier", "--write", "${staged}"],
      "include": [
        "**/*.ts",
        "**/*.tsx",
        "**/*.vue",
        "**/*.js",
        "**/*.jsx",
        "**/*.json",
        "**/*.yml",
        "**/*.css",
        "**/*.scss"
      ]
    }
  ]
}

```

**Comment by alirezanet** [comment]
Hi @maor365scores,
If you're looking to limit the files, it's better to use the include/exclude options to determine which files are processed by your tool, like csharpier. For instance, you could use `Web/CMS/**/*.cs` to target specific files, depending on your project's structure. cwd is running your command from that specific folder which is not what you want if I understand the problem correctly.

**Reply by maor365scores** [reply]
Hi man thanks for the fast reply
So your suggesting ill add this to the include: 
"Include": "Web/CMS/**/*.cs"
Is that right?

**Reply by alirezanet** [reply]
Yeah, I think so and remove the cwd if you don't need it.

**Reply by alirezanet** [reply]
more info here: https://alirezanet.github.io/Husky.Net/guide/task-configuration.html#glob-patterns
